python/graphstorm/model/lm_embed.py

```diff
@@ -258,6 +258,7 @@ def _load_embeddings(self):
         for ntype in self._lm_models.ntypes:
             embed_path = os.path.join(os.path.join(self._embed_path, ntype),
                     self._get_model_name(ntype))
+            embed_path = os.path.join(embed_path, ntype)
```

Comment:
why do we add ntype in the path again?

Reply:
Because, the node embedding saving logic has been changed.
It automatically save embeddings from different ntypes into embedding/<ntype>/ folder.